container/lxd/container.go

```diff
@@ -318,12 +321,27 @@ func (s *Server) RemoveContainer(name string) error {
 		}
 	}
 
-	op, err := s.DeleteContainer(name)
-	if err != nil {
-		return errors.Trace(err)
+	retryArgs := retry.CallArgs{
```

Comment:
Maybe a comment regarding the upstream LXD behaviour/advice?

Reply:
Agreed